user-org-util/src/main/java/org/sunbird/util/ProjectLogger.java

```diff
@@ -164,4 +164,54 @@ private static String getBELog(
     }
     return jsonMessage;
   }
+  /**
+   * This method will be called from Application class(default startup class). which will
+   * automatically set the Log level by taking its value from system environment file.
+   */
+  public static void setUserOrgServiceProjectLogger() {
+
+    String level = getLevelFromEnv();
+    switch (level.toUpperCase()) {
+      case "INFO":
+        changeLogLevel(Level.INFO);
+        break;
+      case "ERROR":
+        changeLogLevel(Level.ERROR);
+        break;
+      case "DEBUG":
+        changeLogLevel(Level.DEBUG);
+        break;
+      case "WARN":
+        changeLogLevel(Level.WARN);
+        break;
+      case "ALL":
+        changeLogLevel(Level.ALL);
+        break;
+      default:
+        changeLogLevel(Level.INFO);
+    }
+  }
+
+  /**
+   * This method will be take Log Level as input params and will set the Log Level dynamically.
+   * Configurator class is in package org.apache.logging.log4j.core.config.
+   *
+   * @param level
+   */
+  public static void changeLogLevel(Level level) {
```

Comment:
add unit test of this method.
1. verify after execution of this method Logger object level need to be reset with passed value.